modules/base/markdown.go

```diff
@@ -250,6 +250,7 @@ func PostProcessMarkdown(rawHtml []byte, urlPrefix string) []byte {
 func RenderMarkdown(rawBytes []byte, urlPrefix string) []byte {
 	result := RenderRawMarkdown(rawBytes, urlPrefix)
 	result = PostProcessMarkdown(result, urlPrefix)
+	Sanitizer.AllowAttrs("class").Matching(regexp.MustCompile(`[\p{L}\p{N}\s\-_',:\[\]!\./\\\(\)&]*`)).OnElements("code")
```

Comment:
The same place where Sanitizer is declared would be my choice. 

Something like: Sanitizer = UGCPolicy.AllowAttr.Match.OnElements

This way we can define rules all at once.